core/src/test/java/org/elasticsearch/common/network/InetAddressesTests.java

```diff
@@ -16,11 +16,13 @@
 
 package org.elasticsearch.common.network;
 
+import org.elasticsearch.common.SuppressForbidden;
 import org.elasticsearch.test.ESTestCase;
 
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 
+@SuppressForbidden(reason = "checks that InetAddresses logic is consistent with InetAddress")
```

Comment:
I'm not happy to suppress a whole class either! In this case I understand wanting to suppress for all those newly forbidden methods across the class but this'll suppress failures for stuff like System.out....